Quick hotfix before project demo

thomaseizinger opened the pull request:
Co-authored-by: Daniel Karzel <daniel@comit.network>

thomaseizinger committed:
Quick hotfix before project demo

Co-authored-by: Daniel Karzel <daniel@comit.network>

da-kami commented on app/components/BalanceHorizontal.tsx:
```diff
@@ -0,0 +1,163 @@
+        setEthBalanceAsCurrencyValue(ethCurrencyValue);
+      } catch (e) {
+        console.error(e);
+        console.warn("Falling back to ETH balance 0.")
```
Dude, there is an unexpected console statement!!!

da-kami committed:
Fix the linter

Changed to use standard react linting and not the pre-configured electron settings.

da-kami committed:
Cleanup unused code and outdated TODOs

da-kami committed:
Change Role to be enum, fix float point conversion in OrderCreator

da-kami committed:
Distinguish bitcoin regtest-wallet by account index and fund correct addresses

da-kami committed:
Add emotions cache to fix error message caused by emotions bug

da-kami committed:
Fix swap step setup typo

da-kami committed:
Update README

da-kami committed:
Close the transport after ledger interaction to avoid error

da-kami commented:
Be green already CI!
